interface/main/finder/dynamic_finder_ajax.php

```diff
@@ -16,6 +16,52 @@
 require_once("../../globals.php");
 require_once($GLOBALS['srcdir']."/options.inc.php");
 
+/**
+ * @param $wordInLang
+ * @return bool
+ */
+function ltox($wordInLang)
+{
+    if (strlen($wordInLang) <= 0) {
+        return false;
+    }
+
+    $sql = "SELECT option_id FROM list_options WHERE list_id = 'sex' AND lower(title) like '".mb_strtolower($wordInLang)."%' or lower(option_id) like '".mb_strtolower($wordInLang)."%' ORDER BY option_id ASC LIMIT 1";
+    $res = sqlStatement($sql);
+    if ($row = sqlFetchArray($res)) {
+        $wordInLang = $row["option_id"];
+    }
+    return $wordInLang;
+}
+
+//Function partiality translate date to need format
+function DateToYYYYMMDD_Partial($DateValue)
+{
+    // Get current display date format
+    $date_display_format = $GLOBALS['date_display_format'];
+    if ($date_display_format > 0) {
+        $date_inp = explode("/", $DateValue);
+
+        $day_input = attr(trim($date_display_format == 1 ? $date_inp[1] : $date_inp[0]));
+        $month_input = attr(trim($date_display_format == 1 ? $date_inp[0] : $date_inp[1]));
+        $year_input = attr(trim($date_inp[2]));
+
+        $day_input = str_pad($day_input, 2, '0', STR_PAD_LEFT);
+        $month_input = (strlen($month_input) > 0 ? str_pad($month_input, 2, '0', STR_PAD_LEFT) . "-" : "");
+        $year_input = (strlen($year_input) > 0 ? $year_input . "-" : "");
+
+        $DateValue = $year_input . $month_input . $day_input;
+    }
+    return $DateValue;
+}
+
+//Function to prepare a string for "Search with exact method" or without it
```

Comment:
This function needs some comments to show how it works (I figured it out, but my brain hurt after :) )

Reply:
Hi @bradymiller . Commented )